releases/2.15/beta/mlflow/bundle.yaml

```diff
@@ -0,0 +1,29 @@
+bundle: kubernetes
+name: mlflow
+docs: https://discourse.charmhub.io/t/mlflow-docs-index/10836
+applications:
+  mlflow-minio:
+    charm: minio
+    channel: ckf-1.9/stable
+    scale: 1
+    trust: true
+    _github_repo_name: minio-operator
+    _github_repo_branch: main
+  mlflow-mysql:
+    charm: mysql-k8s
+    channel: 8.0/stable
+    scale: 1
+    trust: true
+    constraints: mem=2G
```

Comment:
I know maybe this is inherited from other bundles (CKF 1.9 has it, for instance), but I wonder if this constraint is required at all for production environments where this bundle will be deployed. Would be worth investigating. Not a blocker though.